inference/environments/AzureML-minimal-ubuntu20.04-py38-cuda11.6.2-gpu-inference/context/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/azureml/inference-base-gpu-2004:{{latest-image-tag}}
+
+RUN conda create -n minimal \
+    python=3.8.13 pip=22.1.2
+
+ENV AZUREML_CONDA_ENVIRONMENT_PATH /azureml-envs/minimal
```

Comment:
This environment isn't guaranteed to be created in the /azureml-envs/... directory. Please see https://github.com/Azure/azureml-assets/blob/main/training/general/src/environments/pytorch-1.11-ubuntu20.04-py38-cuda11-gpu/context/Dockerfile for the correct way to do this.

Reply:
Or maybe you just need to delete these lines:

```
RUN conda create -n minimal \
    python=3.8.13 pip=22.1.2
```